doc/VitessOverview.md

```diff
@@ -1,22 +1,43 @@
-Vitess is a database solution for scaling MySQL. It's architected to run as
-effectively in a public or private cloud architecture as it does on dedicated
-hardware. It combines and extends many important MySQL features with the
-scalability of a NoSQL database. Vitess has been serving all YouTube database
-traffic since 2011.
+Vitess is a database solution for deploying, scaling and managing large clusters of MySQL instances.
+It's architected to run as effectively in a public or private cloud architecture as it does
+on dedicated hardware. It combines and extends many important MySQL features with the
+scalability of a NoSQL database. Vitess can help you with the following problems:
 
-## Vitess on Kubernetes
+1. Scaling a MySQL database by allowing you to shard it, while keeping application changes to a minimum.
+2. Migrating from baremetal to a private or public cloud.
+3. Deploying and managing a large number of MySQL instances.
 
-[Kubernetes](http://kubernetes.io/) is an open-source orchestration system for Docker containers, and Vitess can run as a Kubernetes-aware cloud native distributed database.
+Vitess includes compliant JDBC and Go database drivers using a native query protocol. Additionally, it implements the MySQL server protocol which is compatible with any other language.
 
-Kubernetes handles scheduling onto nodes in a compute cluster, actively manages workloads on those nodes, and groups containers comprising an application for easy management and discovery.
-This provides an analogous open-source environment to the way Vitess runs in YouTube,
-on the [predecessor to Kubernetes](http://blog.kubernetes.io/2015/04/borg-predecessor-to-kubernetes.html).
+Vitess has been serving all YouTube database traffic since 2011, and has now been adopted by many enterprises for their production needs.
 
-<div style="text-align:center">
-<a class="btn btn-default btn-lg" href="/getting-started/" role="button" style="margin-bottom:16px">
-<img src="/images/kubernetes.svg" style="width:30px;height:30px;margin-top:-5px">
-Quickstart</a>
-</div>
+## Features
+
+* **Performance**
+  * Connection pooling - Scale front-end connections while optimizing MySQL performance.
```

Comment:
IMO this would make more sense as "Multiplex front-end application queries onto a pool of MySQL connections to optimize performance".